### PR TITLE
Update test-configuration-options.md

### PR DESCRIPTION
**Pull request by wswebcreation**
Fix a bug in our docs

### CODE REVIEW

File: docs/dev/test-configuration-options.md [hunk 1]
```diff
@@ -792,15 +792,18 @@ Using Appium 2? Prevent `appium:`-prefix repetitiveness and start using [`appium
 
 <p><small>| OPTIONAL | BOOLEAN | <span className="sauceGreen">Virtual and Real Devices</span> | <span className="sauceGreen">Android Only</span> |</small></p>
 
-Whether to grant all the requested application permissions automatically when a test starts(`true`). `false` by default.
+Whether to grant all the requested application permissions automatically when a test starts(`true`).
```
Comment (lysannep) [inline docs/dev/test-configuration-options.md:795]:
```suggestion
Whether to grant all the requested application permissions automatically when a test starts (`true`).
```

File: docs/dev/test-configuration-options.md [hunk 2]
```diff
@@ -792,15 +792,18 @@ Using Appium 2? Prevent `appium:`-prefix repetitiveness and start using [`appium
 
 <p><small>| OPTIONAL | BOOLEAN | <span className="sauceGreen">Virtual and Real Devices</span> | <span className="sauceGreen">Android Only</span> |</small></p>
 
-Whether to grant all the requested application permissions automatically when a test starts(`true`). `false` by default.
+Whether to grant all the requested application permissions automatically when a test starts (`true`).
```
Comment (lysannep) [inline docs/dev/test-configuration-options.md:795]:
```suggestion
Whether to grant all the requested application permissions automatically when a test starts.
```

**Review by lysannep — APPROVED** [review]

**Comment by lysannep** [comment]
@wswebcreation Thank you!

**Comment by wswebcreation** [comment]
> @wswebcreation Thank you!

Thanks you @lysannep for the feedback and approving it

**Pull request merged**